root/provision-neos.sh

```diff
@@ -10,15 +10,19 @@ else
     /init-xdebug.sh
 
     # Layout default directory structure
+    mkdir -p /data/www-temp
     mkdir -p /data/www
     mkdir -p /data/logs
     mkdir -p /data/tmp/nginx
 
     ###
     # Install into /data/www
     ###
-    cd /data/www
+    cd /data/www-temp
```

Comment:
I have a bit different idea.
Sometimes it could be nice to mount the whole `/data/www` folder (or any of its subparts) for local development, but so the data from the container would aleady be there.

In this case this script could leave the cloned and composer installed files in `/data/www-temp` (or /data/www-provisioned), and in addition to that add a startup script that would check if `/data/www/composer.json` exists, and if not do `mv /data/www-temp/* /data/www/`.

Wdyt?

Reply:
Sorry for my late reply. I had some busy days...
But you are right, I didn't thought of project were you just want to mount everything rather than installing it via Composer. So your approach to check if there is a `/data/www/composer.json` (or maybe just even check if `/data/www` is not empty like `if [ -z "$(ls -A /data/www)" ]; then`...?) seem's to be fine to me.

Reply:
Yep, I like your suggestion.

Reply:
It's of cause the other way around. `if [ -z "$(ls -A /data/www)" ];` actually equals to true, if the directory IS empty. Sorry.

Reply:
Hey! Were you able to take your concept any further?

Reply:
Unfortunately not. I haven't forget about it, but am way too overstrained at my job at the moment. Sorry.

Reply:
No problem, take your time!